src/Lykke.Service.Assets/Cache/DistributedCache.cs

```diff
@@ -2,63 +2,63 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.Extensions.Caching.Distributed;
+using StackExchange.Redis;
 
 namespace Lykke.Service.Assets.Cache
 {
     public class DistributedCache<I, T> where T : class, I
     {
-        private readonly IDistributedCache _cache;
+        private readonly IDatabase _redisDatabase;
         private readonly string _partitionKey;
-        private readonly DistributedCacheEntryOptions _cacheOptions;
+        private readonly TimeSpan _expiration;
 
         public DistributedCache(
-            IDistributedCache cache,
+            IDatabase redisDatabase,
             TimeSpan expiration,
             string partitionKey)
         {
-            _cache = cache;
+            _redisDatabase = redisDatabase;
             _partitionKey = partitionKey;
-            _cacheOptions = new DistributedCacheEntryOptions().SetAbsoluteExpiration(expiration);
+            _expiration = expiration;
         }
 
         private string GetCacheKey(string id)
         {
-            return $":{_partitionKey}:{id}";
+            return $"{_partitionKey}:{id}";
```

Comment:
: removed intentionally?

Reply:
Yes, PartitionKey is now fully specified with prefix ('InstanceName').